appcues/src/main/java/com/appcues/debugger/ui/icons/CloseIcon.kt

```diff
@@ -0,0 +1,31 @@
+package com.appcues.debugger.ui.icons
+
+import androidx.compose.ui.graphics.vector.ImageVector
+
+private var filled: ImageVector? = null
+
+@Suppress("UnusedReceiverParameter", "MagicNumber")
```

Comment:
UnusedReceiverParameter because of "DebuggerIcons.Filled"
and MagicNumber because of all the path values.

Reply:
could be nice to have the rationale in a code comment for later reference on those